bindings/c/Test.cpp

```diff
@@ -137,61 +135,61 @@ TEST_CASE_METHOD(Test, "Parameter-framework c api use") {
         </Subsystem>");
     TmpFile libraries("<?xml version='1.0' encoding='UTF-8'?>\
         <SystemClass Name='test'>\
-            <SubsystemInclude Path='" + system.getPath() + "'/>\
+            <SubsystemInclude Path='" +
+                      system.getPath() + "'/>\
```

Comment:
I agree but I don't know to avoid it. I think that clang-format doesn't like multiline strings.